Documentation/botr/readytorun-format.md

```diff
@@ -225,22 +236,67 @@ additional data determined by the flags.
+|:---------------------------|------:|:--------|
+| ``GCREFMAP_SKIP`` | 0 | Not a GC-relevant entry |
+| ``GCREFMAP_REF`` | 1 | GC reference |
+| ``GCREFMAP_INTERIOR`` | 2 | Pointer to a GC reference |
+| ``GCREFMAP_METHOD_PARAM`` | 3 | Hidden method instantiation argument to generic method |
+| ``GCREFMAP_TYPE_PARAM`` | 4 | Hidden type instantiation argument to generic method |
+| ``GCREFMAP_VASIG_COOKIE`` | 5 | VARARG signature cookie |
+
+The position values are calculated in ``size_t`` aka ``IntPtr`` units (4 bytes for 32-bit architectures vs. 8 bytes for 64-bit architectures) and start at the offset ``TransitionBlock::GetOffsetOfFirstGCRefMapSlot`` relative to the transition frame address. For x86 the position encoding is [somewhat more complicated](https://github.com/dotnet/coreclr/blob/d5d18896900561b7aaf38ba9501a8525a4b9caea/src/vm/frames.cpp#L1326).
+
+* The encoding always starts at the byte boundary. The high order bit of each byte is used to signal end of the encoding stream. The last byte has the high order bit zero. It means that there are 7 useful bits in each byte. 
+
+* "pos" is always encoded as delta from previous pos.
 
-**TODO**: Document the GC info encoding. It is the same encoding as used by NGen. The typical cost is 1 
-byte per slot. This data would not be required for runtimes that allow conservative stack scanning.
+* The basic encoding unit is two bits. Values 0, 1 and 2 are the common constructs (skip single slot, GC reference, interior pointer). Value 3 means that extended encoding follows. 
+
+* The extended information is integer encoded in one or more four bit blocks. The high order bit of the four bit block is used to signal the end.
+
+* For x86, the encoding starts with size of the callee popped stack. The size is encoded using the same mechanism as above (two bit
+basic encoding, with extended encoding for large values).
 
 ## READYTORUN_SECTION_RUNTIME_FUNCTIONS
 
 This section contains sorted array of ``RUNTIME_FUNCTION`` entries that describe all functions in the 
 image with pointers to their unwind info. The standard Windows xdata/pdata format is used.
 ARM format is used for x86 to compensate for lack of x86 unwind info standard.
-The unwind info blob is immediately followed by gc info blob.
+The unwind info blob is immediately followed by GC info blob. The encoding slightly differs for amd64 which encodes an extra 4-byte  representing the end RVA of the frame info blob.
+
+### RUNTIME_FUNCTION (x86, arm, arm64, size = 8 bytes)
+
+| Offset | Size | Value |
+|-------:|-----:|:------|
+| 0 | 4 | Frame info start RVA |
+| 4 | 4 | GC info start RVA |
 
-**TODO**: Document the GC info encoding. It is the same encoding as used by JIT and NGen.
+### RUNTIME_FUNCTION (amd64, size = 12 bytes)
+
+| Offset | Size | Value |
+|-------:|-----:|:------|
+| 0 | 4 | Frame info start RVA |
+| 4 | 4 | Frame info end RVA (1 plus RVA of last byte) |
+| 8 | 4 | GC info start RVA |
```

Comment:
Technically, it is Unwind info start...

Reply:
Fixed in 2nd commit, thanks for keeping an eye on proper terminology, I never get that right.